api/tests/opentrons/protocol_engine/execution/test_labware_movement_handler.py

```diff
@@ -229,9 +234,13 @@ async def test_move_labware_with_gripper(
         current_location=from_location,
         new_location=to_location,
         user_offset_data=user_offset_data,
+        delay_after_drop=delay,
     )
 
     gripper = OT3Mount.GRIPPER
+    decoy.verify(
+        await ot3_hardware_api.do_delay(delay or 0), times=0 if delay is None else 1
+    )
```

Comment:
I don't think the `times=0` here is doing quite what you want.

If the subject code has a bug where it does `.do_delay(1)` unconditionally, this assertion will pass because it *did* call `.do_delay(0)` zero times, which I don't think is your intent.

We want to assert that it called `do_delay(<anything>)` zero times. I would write this like this:

```suggestion

    if delay is None:
        decoy.verify(
            await ot3_hardware_api.do_delay(),  # type:ignore[call-arg]
            times=0,
            ignore_extra_args=True
        )
    else:
        decoy.verify(await ot3_hardware_api.do_delay(delay))

```

Reply:
thx thx thx